posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -553,6 +558,10 @@ public class PostHogReplayIntegration(
             }
         }
 
+        if (view.isComposeView()) {
+            findMaskableComposeRects(view, maskableWidgets)
+        }
```

Comment:
I think all of the checks above and the one below (view.isNoCapture()) can be part of the else branch, so we don't check the other stuff unnecessarily, a view is either a compose view or not.

Reply:
I replaced it with a `when` block. I guess it'll make it a bit more readable too.